db/sql/05_msar.sql

```diff
@@ -4979,6 +4978,10 @@ BEGIN
         END LOOP;
 
         ref_col_id := ref_chain[ref_chain_length];
+        ref_col_name := msar.get_column_name(contextual_tab_id, ref_col_id);
+        IF ref_col_name IS NULL THEN
+          CONTINUE template_parts_loop;
+        END IF;
```

Comment:
Seems redundant, we check whether `ref_col_name IS NOT NULL` after a couple of lines. Please revert this.